aten/src/ATen/native/native_functions.yaml

```diff
@@ -10034,7 +10034,7 @@
   dispatch:
     CPU, CUDA: segment_reduce_kernel
 
-- func: segment_reduce_backward(Tensor grad, Tensor output, Tensor data, *, Tensor? lengths=None) -> Tensor
+- func: segment_reduce_backward(Tensor grad, Tensor output, Tensor data, str reduce, *, Tensor? lengths=None) -> Tensor
```

Comment:
This is BC breaking, must provide default

Reply:
thanks for taking a look @ezyang . Afaik, currently there is no user of this function (internally or externally). Because of that, wanted to keep signature like this.
Given this, would you recommend to make it default or keep with this version?

Reply:
then you must shut up the BC test by whitelisting this op. This op is in prime namespace, if you expect to be continuing to change its semantics you probably should underscore it or something.